.docs/index.md

```diff
@@ -129,53 +129,54 @@ is compatible.
 #### Storage Provider Support
 The following storage providers and platforms are supported by REX-Ray.
 
-| Provider              | Storage Platform  | <center>[Docker](https://docs.docker.com/engine/extend/plugins_volume/)</center> | <center>Containerized</center> |
-|-----------------------|----------------------|:---:|:---:|
-| Amazon EC2 | [EBS](./user-guide/storage-providers/aws.md#aws-ebs) | ✓ | ✓ |
-| | [EFS](./user-guide/storage-providers/aws.md#aws-efs) | ✓ | ✓ |
-| | [S3FS](./user-guide/storage-providers/aws.md#aws-s3fs) | ✓ | ✓ |
-| Ceph | [RBD](./user-guide/storage-providers/ceph.md#ceph-rbd) | ✓ | ✓ |
-| Dell EMC | [Isilon](./user-guide/storage-providers/dellemc.md#dell-emc-isilon) | ✓ | ✓ |
-| | [ScaleIO](./user-guide/storage-providers/dellemc.md#dell-emc-scaleio) | ✓ | ✓ |
-| DigitalOcean | [Block Storage](./user-guide/storage-providers/digitalocean.md#do-block-storage) | ✓ | ✓ |
-| FittedCloud | [EBS Optimizer](./user-guide/storage-providers/fittedcloud.md#ebs-optimizer) | ✓ | |
-| Google | [GCE Persistent Disk](./user-guide/storage-providers/google.md#gce-persistent-disk) | ✓ | ✓ |
-| Microsoft | [Azure Unmanaged Disk](./user-guide/storage-providers/microsoft.md#azure-ud) | ✓ | ✓ |
-| OpenStack | [Cinder](./user-guide/storage-providers/openstack.md#cinder) | ✓ | ✓ |
-| VirtualBox | [Virtual Media](./user-guide/storage-providers/virtualbox.md#virtualbox) | ✓ | |
-
+| Provider        | Storage Platform                                                             | Docker | Containerized |
```

Comment:
Hi @jmaitrehenry,

I used to try and keep MD tables aligned like this. Good on you for doing so, but ultimately it never seems worth it. Ch ch changes will eventually leave things a space off in the raw. Makes my eye twitch :)

Reply:
I think it worth it when you work on the documentation and you can update it without rendering it in the same time because you don't know which content is for which column.
But yes, this complexity the work when you have to change the content. I use an online tool for formatting my tables.